aem-authoring-toolkit-plugin/src/main/java/com/exadel/aem/toolkit/core/handlers/container/TabsHandler.java

```diff
@@ -81,6 +86,16 @@ public void accept(Class<?> componentClass, Element parentElement) {
             Handler.appendToContainer(thisTabFields, tabElement);
             allFields.removeAll(thisTabFields);
         }
+
```

Comment:
This piece of code is neat and proper but seems a bit over-engineering. Looks like we are specially and intentionally hunt for erroneous tab fields.
To my mind, we must move along a more straightforward way.
1) pick up fields for 1st tab (all those with the first tab specified as-is, and those without any tab specified). Remove them from allFields
2) pick up fields for 2... n-th tabs (those with tab explicitly specified); remove from allFiels
3) handle exception for each of those remaining in `allFields`

Please don't take it personal, you are doing really well)